--- rnd.orig.c	2021-05-17 17:48:07.062003577 +0000
+++ rnd.c	2021-05-17 17:49:03.677317067 +0000
@@ -35,6 +35,7 @@
 
 
 /* takes less than 17 nsec on my laptop */
+#if 0
 int parity(unsigned int a) {
     int b;
     int c,d0;
@@ -53,7 +54,13 @@
 	  : "d" (a)
 	);
     return b;
-} 
+}
+#endif
+
+/* WCP Added in for portability on RPi4 */
+int parity(unsigned int a) {
+ return __builtin_parity(a);
+};
 
 /* this is an implementation of an m-sequence */
 
